testsuite/templates/launch.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% block body %}
+    Loading....
+    <script src="/static/fhir-client.js"></script>
+    <script>
+        FHIR.oauth2.authorize({
+            "client_id": "f662d614-8b45-48c2-92f8-c84a20edccc3",
+            "scope": "launch patient/Patient.read patient/Observation.read patient/Condition.read patient/MedicationOrder.read patient/MedicationStatement.read patient/MedicationDispense.read patient/MedicationAdministration.read patient/AllergyIntolerance.read patient/Procedure.read patient/Immunization.read patient/DocumentReference.read"
```

Comment:
A separate page for vendor is probably fine, since in general we **shouldn't** be using this ("EHR Launch") approach to launch anyway. We want to be using the "Standalone Launch". Right now, Cerner only supports EHR Launch, so we need this for Cerner. Maybe there will be a few others too.

Have you thought about how to manage these launch scenarios in cases where >1 user might be running the test suite at the same time?